test/e2e/network/service_latency.go

```diff
@@ -135,17 +135,9 @@ var _ = common.SIGDescribe("Service endpoints latency", func() {
 })
 
 func runServiceLatencies(ctx context.Context, f *framework.Framework, inParallel, total int, acceptableFailureRatio float32) (output []time.Duration, err error) {
-	cfg := testutils.RCConfig{
-		Client:       f.ClientSet,
-		Image:        imageutils.GetPauseImageName(),
-		Name:         "svc-latency-rc",
-		Namespace:    f.Namespace.Name,
-		Replicas:     1,
-		PollInterval: time.Second,
-	}
-	if err := e2erc.RunRC(ctx, cfg); err != nil {
```

Comment:
same as in the other PR, this seems to wait so you need to wait for the deployment to be ready after its creation